makefile

```diff
@@ -22,6 +22,7 @@ install:
 deploy:
 	python setup.py sdist --manifest-only
 	python setup.py sdist --formats zip,gztar upload
+	mkdocs gh-deploy -r pages -b master
```

Comment:
Turns out this doesn't work. It hoses your local master branch the first time and then (after more commits are added) results in merge conflicts. See davisp/ghp-import#65 for details.